app/code/OleksiiBezpoiasnyi/ControllerDemo/view/frontend/templates/info.phtml

```diff
@@ -0,0 +1,4 @@
+<?php /** @var \OleksiiBezpoiasnyi\ControllerDemo\Block\Demo $block */ ?>
+<?php /** @var \Magento\Framework\Escaper $escaper */ ?>
+<p class="oleksiib-controller-demo"><?= $escaper->escapeHtml(__('Full name: %1', $block->getFirstName() . ' ' . $block->getLastName())) ?></p>
```

Comment:
@AlexeyBezpoyasny First name and last name must be separate parameters, not a single one.